### PR TITLE
Remove pragma form of export keyword and update tests

### PR DESCRIPTION
**Pull request by lydia-duncan**
variables/bradc/pragmaMultipleVars.chpl was using the export pragma to
test the application of pragmas in multiple variable declarations.  I updated this
test to use a different pragma ("no doc").  It looked like the futures in
separate_compilation/jturner just hadn't been updated to use the keyword form,
so I updated them and while I was there added .bad files for all of them.
use_array was generating different output than the other tests because it didn't
declare the return type of the function it wanted to use.  The error message is
a different problem, so I modified use_array to work around it and get the same
message most of the other futures were getting.

### CODE REVIEW

**Comment by lydia-duncan** [comment]
@bradcray, would you mind taking a look at test/variables/bradc/pragmaMultipleVars.chpl and letting me know what should be done for it?  My first thought was to replace the use of export with "no doc" or "locale private", something that'll be more long-lived, but the description seemed to include export as part of the expectations, so I wasn't confident.  Would you mind also reviewing the patch? :)

File: test/separate_compilation/jturner/use_array.chpl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-proc dothis(r:[1..2] real); 
+proc dothis(r:[1..2] real): real;
```
Comment (bradcray) [inline test/separate_compilation/jturner/use_array.chpl]: What's this change for?

**Comment by bradcray** [comment]
I think converting pragmaMultipleVars to any other pragma would be fine.  And then the intention is to retire the pragma form of export, right?

**Comment by lydia-duncan** [comment]
That's correct.  I'd include it in this patch, if that's alright

File: test/separate_compilation/jturner/use_array.chpl (hunk 1 above)
Comment (lydia-duncan) [inline test/separate_compilation/jturner/use_array.chpl]: We were getting an earlier complaint about not being able to figure out the return type that's unrelated to the problem this version was trying to expose (separate compilation involving array types gives its own special message).  I have a bug future for that problem elsewhere, so this didn't need to test it

Comment (bradcray) [inline test/separate_compilation/jturner/use_array.chpl]: Oh, I missed that this is a .future.  Would it be possible to add a .bad for it while you're here?

Comment (lydia-duncan) [inline test/separate_compilation/jturner/use_array.chpl]: I think so.  These are all futures, so I can update them all together.

**Pull request merged**